cmd/chart-repo/utils.go

```diff
@@ -98,6 +98,10 @@ func syncRepo(dbSession datastore.Session, repoName, repoURL string, authorizati
 	}
 
 	charts := chartsFromIndex(index, r)
+	if len(charts) == 0 {
+		errors.New("no charts in repository index")
```

Comment:
We'll need to return the error so that the caller can exit appropriately

Reply:
done